lib/tasks/translation.rake

```diff
@@ -85,17 +85,17 @@ namespace :translation do
 
   desc "Add missing translations"
   task(:add_missing, [:locale] => [:environment]) do |t, args|
-    locale = args[:locale] || "en"
+    locale = args[:locale]
     require 'i18n/tasks/cli'
 
-    I18n::Tasks::CLI.start(["add-missing", "-l", locale])
+    locale ? I18n::Tasks::CLI.start(["add-missing", "--nil-value", "-l", locale]) : I18n::Tasks::CLI.start(["add-missing", "--nil-value"])
```

Comment:
This could be [DRY](https://riptutorial.com/ruby-on-rails/example/3932/don-t-repeat-yourself--dry-)'d up by constructing the arguments array in advance:

```ruby
i18n_args = ["add-missing", "--nil-value"]
i18n_args << ["-l", locale] if locale

I18n::Tasks::CLI.start(i18n_args)
```

Could you explain in the commit message why we need the `--nil-value` flag?

Reply:
@ChrisBAshton I have applied it to add_missing task, and left normalize task unchanged (it doesn’t have  "--nil-value" flag so is more readable than add_missing).